src/containers/Identity/PersonalInfo/View.js

```diff
@@ -72,17 +72,30 @@ const PersonalInfo = (props) => {
 
   const updateSearch = (searchValue) => {
     const newData = claimCategories.filter((claimCategory) => {
```

Comment:
Please add a comment what this function is doing and how. Think about moving it into a util function just pass into it whatever it needs.

Reply:
done